picard/file_dilaog.py

```diff
@@ -0,0 +1,274 @@
+                self.recu_folder(self.send_things,os.path.join(self.path,hola))
+            else:
+                self.send_things.append(os.path.join(self.path,hola))
+        for  i in self.send_things:
+            print(i)
+        #print(self.send_things)
+        sys.exit()
+
+    def recu_folder(self,lis,path):
+       temp = os.listdir(path)
+       for i in temp:
+            if os.path.isfile(os.path.join(path,i)):
+                for j in self.formats_lower:
+                    if i.endswith('.'+j):
+                        lis.append(os.path.join(path,i))
+            else:
+                self.recu_folder(lis,os.path.join(path,i))
+
+    def moveBack_action(self):
+        self.path = os.path.dirname((os.path.abspath(self.path)))
+        self.show_files(False)
+
+    def moveNext_action(self):
+        indexes = self.table.selectionModel().selectedRows()
+        if len(indexes) == 1:
+            if os.path.isdir(os.path.join(self.path,self.table.item(indexes[0].row(),0).text()[6:])):
+                self.path = os.path.join(self.path,self.table.item(indexes[0].row(),0).text()[6:])
+                self.show_files(False)
+        if len(indexes) == 0:
+            temp_name = []
+            for i in os.listdir(self.path) :
+                if os.path.isdir(os.path.join(self.path,i)):
+                    temp_name.append(i)
+            if len(temp_name) >0:
+                #sort(temp_name)
+                self.path=os.path.join(self.path,temp_name[0])
+                self.show_files(False)
+
+    def show_files(self,hidden):
+        self.hidden_items = 0
+        self.files_to_show()
+        if hidden == False :
+            self.table.setRowCount(len(self.dirs)+len(self.files)-self.hidden_items)
+        else:
+            self.table.setRowCount(len(self.dirs)+len(self.files))
+        start,pos,j=0,0,0
+        while start < len(self.dirs):
+            if hidden == False :
+                if Is_hidden(self.dirs[start]) == True:
+                    start=start+1
+                    continue
+            self.table.setItem(pos,0,QTableWidgetItem("      "+self.dirs[start]))
+            self.table.setItem(pos,1,QTableWidgetItem(self.dirs_size[start]))
+            self.table.setItem(pos,2,QTableWidgetItem(self.dirs_modi_time[start]))
+            self.table.setCellWidget(pos, 0, ImgWidget1("../resources/img-src/folder24_24.png"))
+            pos,start=pos+1,start+1
+
+        while j < len(self.files):
+            if hidden == False :
+                if Is_hidden(self.files[j]) == True:
+                    j=j+1
+                    continue
+            self.table.setItem(pos,0,QTableWidgetItem("      "+self.files[j]))
+            self.table.setItem(pos,1,QTableWidgetItem(self.files_size[j]))
+            self.table.setItem(pos,2,QTableWidgetItem(self.files_modi_time[j]))
+            self.table.setCellWidget(pos, 0, ImgWidget1("../resources/img-src/music24_24.png"))
```

Comment:
Compile your resources properly. We use a qrc file to get icons. See picard/resources on how to do this.